src/arrayMethodJoin.js

```diff
@@ -4,8 +4,21 @@
  * Implement method join
  */
 function applyCustomJoin() {
-  [].__proto__.join2 = function(separator) {
-    // write code here
+  [].__proto__.join2 = function(separator = ',') {
+    const internalSeparator = separator;
```

Comment:
there is no need in this variable. you can use `separator` parameter, that is passed to the function, directly